airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -224,6 +224,38 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  SelectiveAuthenticator:
+    title: Selective Authenticator
+    description: Authenticator that selects concrete authenticator based on config property.
+    type: object
+    additionalProperties: true
+    required:
+      - type
+      - authenticators
+      - authenticator_selection_path
```

Comment:
this field is not defined. can you also add a testcase to `test_model_to_component_factory.py`?

Reply:
done